test/distributed/test_distributed.py

```diff
@@ -678,6 +690,27 @@ def test_broadcast_cuda(self):
         rank_to_GPU = self._init_multigpu_helper()
         self._test_broadcast_helper(group, group_id, rank, True, rank_to_GPU)
 
+    @unittest.skipIf(
+        BACKEND != "nccl",
+        "Only Nccl backend supports stream",
+    )
+    @skip_if_no_gpu
+    def test_nccl_high_priority_stream(self):
```

Comment:
@mrshenli I added one test here. Do you have suggestions about other tests to add?